### PR TITLE
Update perf and scalability page for istio 1.4 release

### PR DESCRIPTION
**Pull request by carolynhu**
this PR is on hold.

### CODE REVIEW

**Comment by carolynhu** [comment]
cc @mandarjog we also need to update perf number for telemetryv2

**Comment by howardjohn** [comment]
The svg looks corrupted

**Comment by carolynhu** [comment]
@howardjohn this is just a temporary file. I will fix it after we collect more perf data.

**Comment by geeknoid** [comment]
@mandarjog @carolynhu 1.4 is due tomorrow, do we have time to update this stuff?

**Comment by carolynhu** [comment]
yes I will update it today

Martin Taillefer <notifications@github.com>于2019年11月11日 周一上午6:09写道：

File: content/en/docs/ops/performance-and-scalability/index.md [hunk 1]
```diff
@@ -34,10 +34,10 @@ The [Istio load tests](https://github.com/istio/tools/tree/{{< source_branch_nam
 of **1000** services and **2000** sidecars with 70,000 mesh-wide requests per second.
 After running the tests using Istio {{< istio_release_name >}}, we get the following results:
 
-- The Envoy proxy uses **0.6 vCPU** and **50 MB memory** per 1000 requests per second going through the proxy.
+- The Envoy proxy uses **0.5 vCPU** and **33 MB memory** per 1000 requests per second going through the proxy.
```
Comment (howardjohn) [inline content/en/docs/ops/performance-and-scalability/index.md]: Where does 33mb come from?

**Comment by carolynhu** [comment]
@howardjohn Please take another look

File: content/en/docs/ops/performance-and-scalability/index.md [hunk 2]
```diff
@@ -127,8 +127,8 @@ This will decrease the amount data flowing through the system, which will in tur
 - `baseline` Client pod directly calls the server pod, no sidecars are present.
 - `server-sidecar` Only server sidecar is present.
 - `both-sidecars` Client and server sidecars are present. This is a default case inside the mesh.
-- `nomixer-both` Same as **both-sidecars** without Mixer. `MixerV2` latency profile will be similar.
-- `nomixer-server` Same as **server-sidecar** without Mixer. `MixerV2` latency profile will be similar.
+- `nomixer-both` Same as **both-sidecars** without Mixer. `TelemetryV2` latency profile will be similar.
+- `nomixer-server` Same as **server-sidecar** without Mixer. `TelemetryV2` latency profile will be similar.
```
Comment (howardjohn) [inline content/en/docs/ops/performance-and-scalability/index.md]: should we add the "telemetry v2" option as part of the list

File: content/en/docs/ops/performance-and-scalability/index.md [hunk 3]
```diff
@@ -127,8 +127,8 @@ This will decrease the amount data flowing through the system, which will in tur
 - `baseline` Client pod directly calls the server pod, no sidecars are present.
 - `server-sidecar` Only server sidecar is present.
 - `both-sidecars` Client and server sidecars are present. This is a default case inside the mesh.
-- `nomixer-both` Same as **both-sidecars** without Mixer. `MixerV2` latency profile will be similar.
-- `nomixer-server` Same as **server-sidecar** without Mixer. `MixerV2` latency profile will be similar.
+- `nomixer-both` Same as **both-sidecars** without Mixer. `TelemetryV2` latency profile will be similar.
```
Comment (mandarjog) [inline content/en/docs/ops/performance-and-scalability/index.md]: 1. Pl Add telemetry v2 option, mention that target is no-mixer both.
2. Remove the telemetry v2 mention from this and next bullet.

File: content/en/docs/ops/performance-and-scalability/index.md (hunk 2 above)
Comment (howardjohn) [inline content/en/docs/ops/performance-and-scalability/index.md]:
```suggestion
- `baseline` Client pod directly calls the server pod, no sidecars are present.
- `server-sidecar` Only server sidecar is present.
- `both-sidecars` Client and server sidecars are present. This is a default case inside the mesh.
- `nomixer-both` Same as **both-sidecars** without Mixer.
- `nomixer-server` Same as **server-sidecar** without Mixer.
- `telemetryv2-nullvm_both` Same as **both-sidecars** but with telemetry v2. This is targeted to perform the same as "No Mixer" in the future.
- `telemetryv2-nullvm_serveronly` Same as **server-sidecar** but with telemetry v2. This is targeted to perform the same as "No Mixer" in the future.

**Pull request merged**